entrypoint.sh

```diff
@@ -61,8 +69,10 @@ if [[ $# -gt 0 ]]; then # Headless mode
         echo -e "\t --security-group SECURITY_GROUP"
         echo -e "\t --slack-webhook-url-ssm AWS_SSM_PARAMETER_FOR_SLACK_WEBHOOK_URL"
         echo -e "\t --slack-channel SLACK_CHANNEL"
-        echo "Optional argument:"
+        echo "Optional arguments:"
         echo -e "\t --slack-username SLACK_USERNAME (default: RDSAlert)"
+        echo -e "\t --rds-preferred-name RDS_INSTANCE_PREFERRED_NAMES (example: Production_DB,Preprod_DB)"
+        echo -e "\t --cpu-alert-threshold SLACK_USERNAME (default: 30 per cent)"
```

Comment:
This should say CPU_PERCENTAGE not SLACK_USERNAME :)

Reply:
Fixed in the next commit. Thanks!